src/main/java/io/seqera/tower/cli/commands/pipelines/AbstractPipelinesCmd.java

```diff
@@ -58,6 +63,21 @@ protected PipelineDbDto fetchPipeline(PipelineRefOptions pipelineRefOptions, Lon
         return api().describePipeline(pipelineId, List.of(attributes), wspId, null).getPipeline();
     }
 
+    private static String quotePipelineName(String pipelineName) {
+
+        if (pipelineName == null) return null;
+        if (pipelineName.isEmpty()) return "\"\"";
+
+        // replace '"' with '\"'
+        String escaped = pipelineName.replace("\"", "\\\"");
+
+        if (escaped.startsWith("\\\"") && escaped.endsWith("\\\"")) {
```

Comment:
what happens if only one side of the name has quotes?
Like `"pipeline` or `pipeline"`?